widget/card.go

```diff
@@ -0,0 +1,204 @@
+package widget
```

Comment:
Considering calling this `cardcontainer.go` for consistency with the other container widgets?

Reply:
Fixed - the refactor missed that for some reason - it was originally called Card.